### PR TITLE
Improve wording for the type keyword in the spec

### PR DESCRIPTION
**Pull request by jdesrosiers**
Fixes https://github.com/json-schema-org/json-schema-spec/issues/1080

I adjusted the wording suggested in the issue, so let me know if anyone thinks it doesn't read clearly enough.

### CODE REVIEW

File: jsonschema-validation.xml [hunk 1]
```diff
@@ -225,8 +225,11 @@
                         or "integer" which matches any number with a zero fractional part.
                     </t>
                     <t>
-                        An instance validates if and only if the instance is in any of the sets listed
-                        for this keyword.
+                        If the value of "type" is a string, then an instance validates successfully if
+                        it's type matches the type represented by the value of the string.
+
+                        If the value of "type" is an array, then an instance validates successfully if
+                        it's type matches any of the types indicated by the strings in the array.
```
Comment (karenetheridge) [inline jsonschema-validation.xml:232]:
```suggestion
                        its type matches the type represented by the value of the string.

                        If the value of "type" is an array, then an instance validates successfully if
                        its type matches any of the types indicated by the strings in the array.
```

**Review by karenetheridge — CHANGES_REQUESTED** [review]

File: jsonschema-validation.xml (hunk 1 above)
Comment (karenetheridge) [reply jsonschema-validation.xml:232]: ..otherwise, LGTM.

**Review by karenetheridge — CHANGES_REQUESTED** [review]

File: jsonschema-validation.xml (hunk 1 above)
Comment (jdesrosiers) [reply jsonschema-validation.xml:232]: Where's the embarrassment emoji :facepalm:

**Review by gregsdennis — APPROVED** [review]
This is clearer to me and correct.

**Review by awwright — APPROVED** [review]
Much improve, wow

**Pull request merged**